hooks/pre-commit

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+from __future__ import print_function
+from future import standard_library
+standard_library.install_aliases()
+import os
+import urllib.request
+import urllib.parse
+import urllib.error
+from distutils.version import LooseVersion
+import re
+import sys
+import subprocess
+
+pattern = re.compile(r'[."]version["\s":=]+(\d+\.\d+.\d+)')
+version_dict = {}
+version_file = urllib.request.urlopen(
+    "https://raw.githubusercontent.com/weecology/retriever/master/version.txt")
+version_file.readline()
+
+
+def to_string(value):
+    if sys.version_info >= (3, 0, 0):
+        return value.decode("UTF-8")
+    else:
+        return value
+
+
+for lines in version_file:
+    key_values = to_string(lines).split(",")
+    version_dict[key_values[0]] = key_values[1]
+
+
+def get_script_version(file_name):
+    file_name = os.path.split(file_name)[1]
```

Comment:
I'm fine either way, but I tend to prefer `os.path.basename(file_name)` instead of `split` with the index. I like it because I know right away which part of the file name the code is using and don't have to read to and think about the index.